test/kubemark/run-e2e-tests.sh

```diff
@@ -33,11 +33,13 @@ export KUBECONFIG="${ABSOLUTE_ROOT}/test/kubemark/resources/kubeconfig.kubemark"
 export E2E_MIN_STARTUP_PODS=0
 
 if [[ -z "$@" ]]; then
-	ARGS='--ginkgo.focus=\[Feature:Performance\]'
+	ARGS='--ginkgo.focus=[Feature:Performance]'
 else
 	ARGS=$@
 fi
 
+ARGS=$(echo $ARGS | sed 's/\[/\\\[/g' | sed 's/\]/\\\]/g')
```

Comment:
This wouldn't work if you take `ARGS` from the else above right? Because we pass it with the backslashes for all our CI tests. We need to change all of them if we change this I guess.